README.md

```diff
@@ -17,7 +17,9 @@ monitor allows you to observe the device output.
   
   [<img alt='bbc micro:bit' src='https://raw.githubusercontent.com/microsoft/vscode-python-devicesimulator/dev/assets/readmeFiles/microbit/microbit.png'>](#bbc-microbit-simulator)
 
-
+- [**Adafruit CLUE**](#adafruit-clue-simulator) (hidden behind preview flag)
+  
+  [<img alt='Adafruit CLUE' src='assets/readmeFiles/clue/clue.jpg'>](#adafruit-clue-simulator)

```

Comment:
The other pictures have an absolute link. I think it might be connected with the marketplace having issues otherwise, maybe not.
Either way I would make it consistent with the others.

Reply:
I talked to Vandy about this and he said that he had to make two PRs to resolve this (one for relative img links and another one for absolute links)

Reply:
Thanks for checking!
I would assume that since we are checking in the images in this commit, once it's pushed to the appropriate branch the images should display correctly. What do you think?

Reply:
Done :D